app/src/main/res/layout/subscriber_profile_item.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/subscriber_profile_item_layout_height"
+    android:orientation="vertical"
+    android:tag="@string/subscriber_details_profile_layout_tag">
+
+    <LinearLayout
```

Comment:
You can easily recreate this block using a relative layout with an image view and three textviews. Think in the performance when creating the UI